RecoTracker/SpecialSeedGenerators/python/cosmicCDC_cff.py

```diff
@@ -0,0 +1,46 @@
+import FWCore.ParameterSet.Config as cms
+
+# seeding
+from RecoMuon.CosmicMuonProducer.cosmicMuons_cfi import *
+import RecoMuon.MuonIdentification.muons1stStep_cfi
+muonsForCosmicCDC = RecoMuon.MuonIdentification.muons1stStep_cfi.muons1stStep.clone(
+    inputCollectionLabels = cms.VInputTag("cosmicMuons"),
+    inputCollectionTypes = cms.vstring('outer tracks'),
+    fillIsolation = cms.bool(False),
+    fillGlobalTrackQuality = cms.bool(False),
+    fillGlobalTrackRefits = cms.bool(False),
+)
+muonsForCosmicCDC.TrackExtractorPSet.inputTrackCollection = cms.InputTag("cosmicMuons")
+import RecoTracker.SpecialSeedGenerators.outInSeedsFromStandaloneMuons_cfi
+import TrackingTools.KalmanUpdators.Chi2MeasurementEstimator_cfi 
+hitCollectorForOutInMuonSeeds = TrackingTools.KalmanUpdators.Chi2MeasurementEstimator_cfi.Chi2MeasurementEstimator.clone(
+    ComponentName = cms.string('hitCollectorForOutInMuonSeeds'),
+    MaxChi2 = cms.double(100.0), ## was 30 ## TO BE TUNED
+    nSigma  = cms.double(4.),    ## was 3  ## TO BE TUNED 
+    MaxDisplacement = cms.double(100),
+    MaxSagitta = cms.double(-1.0),
+    MinimalTolerance = cms.double(0.5),
+    appendToDataLabel = cms.string(''),
+)
+cosmicCDCSeeds = RecoTracker.SpecialSeedGenerators.outInSeedsFromStandaloneMuons_cfi.outInSeedsFromStandaloneMuons.clone(
```

Comment:
I thought that the first "C" in "CDC" is already for cosmic.
If so "cosmicCDC" is partly tautological
Naming should be somewhat meaningful, non-repetitive.
maybe "cosmic" prefix should be dropped.